### PR TITLE
[SWPWA-370]  Category disabling support

### PR DESCRIPTION
**Pull request by dmitrijs-voronovs**
* 404 page when category is disabled

Depends on https://github.com/scandipwa/catalog-graphql/pull/39

### CODE REVIEW

File: src/app/route/CategoryPage/CategoryPage.container.js [hunk 1]
```diff
@@ -54,7 +55,10 @@ export const mapDispatchToProps = dispatch => ({
         ? BreadcrumbsDispatcher.updateWithCategory(breadcrumbs, dispatch)
         : BreadcrumbsDispatcher.update([], dispatch)),
     requestProductListInfo: options => ProductListInfoDispatcher.handleData(dispatch, options),
-    updateLoadStatus: isLoading => dispatch(updateInfoLoadStatus(isLoading))
+    updateLoadStatus: isLoading => dispatch(updateInfoLoadStatus(isLoading)),
+    updateNoMatch: (options) => {
+        NoMatchDispatcher.updateNoMatch(dispatch, options);
+    }
```
Comment (alfredsgenkins) [inline src/app/route/CategoryPage/CategoryPage.container.js]: I think `() => {}` no needed, you might escape curly brackets.

**Comment by alfredsgenkins** [comment]
If comment in catalog is resolved - and this small thing here + dependency to release of catalog added => we merge.

**Comment by alfredsgenkins** [comment]
Please add composer dep to: https://github.com/scandipwa/catalog-graphql/releases/tag/2.6.4

**Comment by dmitrijs-voronovs** [comment]
Ready!

**Pull request merged**